lib/natalie/compiler/binding_gen.rb

```diff
@@ -901,6 +901,7 @@ def generate_name
 gen.binding('String', 'to_str', 'StringObject', 'to_str', argc: 0, pass_env: false, pass_block: false, return_type: :Object)
 gen.binding('String', 'to_sym', 'StringObject', 'to_sym', argc: 0, pass_env: true, pass_block: false, return_type: :Object)
 gen.binding('String', 'upcase', 'StringObject', 'upcase', argc: 0, pass_env: true, pass_block: false, return_type: :Object)
+gen.binding('String', 'empty', 'StringObject', 'empty', argc: 0, pass_env: true, pass_block: false, return_type: :bool)
```

Comment:
```suggestion
gen.binding('String', 'empty?', 'StringObject', 'is_empty', argc: 0, pass_env: false, pass_block: false, return_type: :bool)
```

You can bind directly to the existing method instead of creating a new one.